monitoring/uss_qualifier/suites/faa/uft/design/README.md

```diff
@@ -0,0 +1,79 @@
+# UFT Message Signing tests for SCD using Uss_Qualifier
+
+UTM Field Test (UFT) has a requirement for USSes to sign their message requests
+and responses in the SCD flow using IATF certificates provided by FAA.
+
+This test suite helps to test the message signing by extending [suites.astm.utm.f3548_21](../../../astm/utm/f3548_21.yaml)
+with scenarios that trigger reporting of message signing in the interactions with mock_uss.
+
+## Test Harness Setup
+
+The test setup includes running the following components on your local machine, as also shown in the [diagram](./InterUss%20Test%20Harness%20With%20Message%20Signing.png)
+
+1. Uss Qualifier - This is the test driver that injects the operations for the SCD flow tests in USSes.
+2. Dummy Auth - This is a dummy auth server that provides tokens for uss-to-uss and uss-to-dss communication. Runs as a dockerized container at port 8085.
+3. DSS - This is a local DSS running in dockerized container, and is available at port 8082.
+4. Mock USS - This is a mock implementation of a USS running as a dockerized container at port 8074. The message signing reports are generated using mock_uss.
+5. USS-under-Test - This is the USS that needs to be tested.
+6. Uss Qualifier Interface. USSes need to develop an interface for their USS
```

Comment:
Looking at the diagram, I think the concept of Test Harness is a bit different for other automated testing interfaces we have been developing. 
In this repository a `Test Harness` is a service implementing the test interface and responsible to map the test driver requests into the USSP proprietary systems. This service is implemented and provided by the USS under test.

You can see an example here for ![Remote Identification](https://github.com/interuss/automated_testing_interfaces/blob/main/rid/automated_rid_testing.png?raw=true).

So on the diagram above, as far as I understand, I would recommend to rename the purple box to `test harness`.

Reply:
I agree that the term "test harness" is being used in a different way in this document; I'd recommend simply renaming instances "Test harness" in this document to "Test setup" -- so, for instance, this:

```
## Test Harness Setup

The test setup includes [...]
```

...would simply change to this:

```
## Test Setup

The test setup includes [...]
```

...which is what the body text already says any way.

Reply:
I can change that.